modules/introduction/partials/new-features-76.adoc

```diff
@@ -209,6 +209,22 @@ The Query service adds new cluster-level, node-level, and request-level paramete
 
 * The SORT BY and GROUP BY operations overspill to disk if they exceed the Query service memory quota.
 
+=== Eventing Service
+
+* The optional parameter `{ "self_recursion": true }` can be used with the INSERT, UPSERT, and REPLACE advanced operations to prevent the suppression of recursive source bucket mutations.
+For more information, see xref:server:eventing:eventing-advanced-keyspace-accessors.adoc#optional-params-recursion[Optional `{ "self_recursion": true }` Parameter].
+
+* The built-in `ANALYTICS()` function allows the Eventing Service to integrate directly with {sqlpp} Analytics.
+This integration simplifies Eventing code logic and lets Eventing benefit from the high availability and load balancing of {sqlpp} Analytics.
+For more information, see xref:server:eventing:eventing-language-constructs.adoc#analytics_call[`ANALYTICS()`].
```

Comment:
Suggest add "Function Call" to the name as the orange formatting makes the underline impossible to see

Reply:
Good call - added function call!